lib/helpers.js

```diff
@@ -16,7 +16,7 @@ function sriDigest(file, fromString) {
 }
 
 function selectedTheme(config, selected) {
-    if (typeof selected === 'undefined' || selected === 'undefined') {
```

Comment:
I think I found the relevant part https://github.com/MaxCDN/bootstrap-cdn/blob/develop/views/_partials/header.pug#L1

This must have changed in pug and they don't pass an empty string IIRC. Is there any way you could check #818 console log for any errors while browsing the site and switching themes?